### DISCUSSION TITLE
Max size of file for file picker (QFile)

### DISCUSSION THREAD

**Discussion by artemklishch** [quasarframework/quasar#16598] [General - Components / Directives / etc] [answered]
Hello!

Is it possible to set some number of values to the property 'max-file-size' in the 'QFile' component?
I nave to upload files of different types to the single input 'QFile' and have different max file size values.

Thank you!

**Comment by metalsadman** [comment, marked as answer]
max-file-size is for all individual files, what you probably have to do is check file parameters on @added event and choose to add or not based on the file size and type.

**Reply by artemklishch** [reply]
Thank you so much!
